addons/auth_ldap/migrations/11.0.1.0/post-migration.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2020 Tecnativa - Jairo Llopis
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env):
```

Comment:
```suggestion
def migrate(env, version):
```
and you can squash commits